core/v1/meshfederation_types.go

```diff
@@ -80,14 +80,16 @@ type MeshFederationSecurity struct {
 	// +required
 	TrustDomain string `json:"trustDomain,omitempty"`
 
-	// Name of secret containing certificate chain to be used to validate
-	// the remote.  This is also used to validate certificates used by the
-	// remote services (both client and server certificates).
-	// XXX: maybe this is only used to initiate a connection, with the actual
-	// certs stored in the status field, as retrieved from the remote mesh, or
-	// maybe this identifies an endpoint used to retrieve a cert chain, a la jwks
+	// Name of ConfigMap containing certificate chain to be used to
+	// validate the remote.  This is also used to validate certificates used by
+	// the remote services (both client and server certificates).  The name of
+	// the entry should be of the form: <federation-name>-ca.crt, e.g. if the
```

Comment:
why the name requirement? Are we planning to support storing multiple root certs in the same ConfigMap? If not, I think just using a generic name like `root-cert.pem` is fine

Reply:
Good question.  I don't have a strong opinion one way or the other.  I didn't give a lot of thought to the user workflow, other than thinking maybe having a single resource with the CAs for all federated clusters would be nice, as opposed to individual resources for each CA.  Having a generic name seems simpler for the user (no need to figure out what name to use).  Should I make the entry key configurable as well?